rum_angular/manifest.json

```diff
@@ -12,13 +12,10 @@
     "title": "Angular",
     "media": [],
     "classifier_tags": [
-      "Supported OS::Any",
-      "Category::Monitoring", 
-      "Category::Network", 
-      "Category::Exceptions", 
-      "Category::Metrics", 
-      "Category::Alerting", 
-      "Category::Issue Tracking"
+      "Category::Metrics",
+      "Category::Network",
+      "Category::Tracing",
```

Comment:
Do you have a category for Browser monitoring? 
Network doesn't feel super related to angular and the mobile app integrations have "Mobile"

Reply:
Agree on removing network here. We don't have a browser monitoring category at this time